pkgs/development/compilers/go/1.12.nix

```diff
@@ -30,11 +30,11 @@ in
 
 stdenv.mkDerivation rec {
   pname = "go";
-  version = "1.12.9";
+  version = "1.13";
```

Comment:
I think we should keep both 1.12 and 1.13 around for a while, as both considered stable upstream at the moment. If we really want to keep only 1.13, then we should at least reflect that in the file name (it's currently called 1.12.nix), and we should also rename the attributes `go_1_12` and `buildGo112*` in pkgs/all-packages/top-level.nix.